detection_rules/packaging.py

```diff
@@ -129,12 +130,33 @@ def _package_notice_file(save_dir):
         with open(NOTICE_FILE, 'rt') as f:
             notice_txt = f.read()
 
-        with open(os.path.join(save_dir, 'notice.ts'), 'w') as f:
+        with open(os.path.join(save_dir, 'notice.ts'), 'wt') as f:
             commented_notice = [' * ' + line for line in notice_txt.splitlines()]
             lines = ['/* eslint-disable @kbn/eslint/require-license-header */', '', '/* @notice']
             lines = lines + commented_notice + [' */', '']
             f.write('\n'.join(lines))
 
+    def _package_index_file(self, save_dir):
+        """Convert and save index file with package."""
+        sorted_rules = sorted(self.rules, key=lambda k: (k.metadata['creation_date'], os.path.basename(k.path)))
+        comments = [
+            '// Auto generated file from either:',
+            '// - scripts/regen_prepackage_rules_index.sh',
+            '// - detection-rules repo using CLI command build-release',
+            '// Do not hand edit. Run script/command to regenerate package information instead',
+            ''
+        ]
+        rule_imports = [f"import rule{i} from './{os.path.splitext(os.path.basename(r.path))[0] + '.json'}';"
+                        for i, r in enumerate(sorted_rules, 1)]
+        const_exports = ['export const rawRules = [']
+        const_exports.extend(f"  rule{i}," for i, _ in enumerate(sorted_rules, 1))
+        const_exports.append("];")
+        const_exports.append(" ")
+
+        index_ts = [JS_LICENSE + '\n'] + comments + rule_imports + const_exports
```

Comment:
nit: add a newline before the  `export const`
```suggestion
        index_ts = [JS_LICENSE]
        index_ts.append("")
        index_ts.append("")
        index_ts.extend(comments)
        index_ts.append("")
        index_ts.extend(rule_imports)
        index_ts.append("")
        index_ts.extend(rule_imports)
```